src/blocks/homepage-articles/edit.js

```diff
@@ -564,7 +564,7 @@ class Edit extends Component {
 		return (
 			<Fragment>
 				<div
-					className={ classes }
+					className={ classes + ' wpnbha' }
```

Comment:
You can use `classnames` for this instead of concatenation. Just add `'wpnbha'` to the list of classes at line 493 and return this to what it was. 

```
const classes = classNames( className, {
			'is-grid': postLayout === 'grid',
			'show-image': showImage,
			[ `columns-${ columns }` ]: postLayout === 'grid',
			[ `type-scale${ typeScale }` ]: typeScale !== '5',
			[ `image-align${ mediaPosition }` ]: showImage,
			[ `image-scale${ imageScale }` ]: imageScale !== '1' && showImage,
			[ `image-shape${ imageShape }` ]: imageShape !== 'landscape',
			'has-text-color': textColor.color !== '',
			'show-caption': showCaption,
			'show-category': showCategory,
			'wpnbha',
		} );
```

```suggestion
					className={ classes }
```

Reply:
Nice! Fixed in f1e497c